.gitignore

```diff
@@ -1,4 +1,5 @@
 _state*
 .idea*
 .DS_Store
-bootstrap/bootstrap.go
\ No newline at end of file
+bin/*
+bootstrap/bootstrap.go
```

Comment:
it's already back in the `master` branch right?  this will just make sure people don't add changes to it to their commits if that do a `git add .` rather than remove it again  ...  unless we want that behavior?